server/src/test/java/org/elasticsearch/rest/action/admin/indices/RestResizeHandlerTests.java

```diff
@@ -70,10 +70,8 @@ private void runTestResizeCopySettingsDeprecated(
             assertThat(e, hasToString(containsString("parameter [copy_settings] can not be explicitly set to [false]")));
         } else {
             handler.prepareRequest(request, mock(NodeClient.class));
-            if (copySettings == null) {
-                assertWarnings(
-                        "resize operations without copying settings is deprecated; "
-                                + "set parameter [copy_settings] to [true] for future default behavior");
+            if ("true".equals(copySettings)) {
```

Comment:
should this be expanded to include when `copySettings` is `""`?

Reply:
Thanks @talevy I pushed 9916d9ca30c6aa7b0ddd0cc177a67e5f4e3e5f7f.